data_structures/linked_list/middle_element_of_linked_list.py

```diff
@@ -13,6 +13,16 @@ def push(self, new_data:int) -> int:
         self.head = new_node
 
     def middle_element(self) -> int:
+        '''
+        >>> link = LinkedList()
+        >>> link.push(4)
+        >>> link.push(3)
+        >>> link.push(8)
+        >>> link.push(7)
+        >>> link.push(9)
+        >>> link.push(11)
+        >>> link.middle_element()
```

Comment:
After this line you need to put the value (without the >>>) that you expect link.middle_element() to return.  This is the test.  If `link.middle_element()` returns that value then the test will pass.

Reply:
There is no `return` statement in `LinkedList.push()` so it does not currently return anything.